frontend/src/components/displays/header.js

```diff
@@ -0,0 +1,43 @@
+import React, { useContext } from 'react';
+
+import { AuthContext } from '../../data/authContext';
+import {headerRoutes} from '../../data/routes'
+import { fire } from '../../data/firebase';
+
+import HeaderLink from './headerLink'
+
+import { Button } from '@material-ui/core';
+
+
+
+
+const Header = () => {
+    const { currentUser } = useContext(AuthContext);
+
+    const signout = () => {
+        fire.auth().signOut();
+    }
+
+    return(
+        <div className='header'>
+            <h1><a href='/'>Arch City Expungement Clinic</a></h1>
+            <div align='center'>
+                {headerRoutes.map((route) => (
+                    <HeaderLink 
+                    activeOnlyWhenExact={true}
+                    to={route.path}
+                    label={route.label}
+                    />
+                ))}
+
+                <div className='loginbtn'>
+                    {currentUser ? (<Button onClick={signout}>Sign Out</Button>) : (<Button href='/login'>Login</Button>)}
```

Comment:
I found this button almost invisible on the header. The only reason I found it was because I saw it mentioned in the code.